embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -35,7 +35,12 @@ internal sealed class EmbType(type: String, subtype: String?) : TelemetryType {
 
         internal object Breadcrumb : System("breadcrumb")
 
-        internal object Log : System("log")
+        internal object Log : System("log") {
+            /**
+             * Attribute name for a unique id identifying the log
+             */
+            val embLogId = EmbraceAttributeKey("log_id")
```

Comment:
What about the other attributes?

Reply:
They are common attributes that non-embrace logs will use as well so I put them in EmbraceAttributeKeys.kt